lms/templates/courseware/info.html

```diff
@@ -10,6 +10,7 @@
 <%block name="headextra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
+<script type="text/javascript" src="${static.url('js/vendor/backbone-min.js')}"></script>
```

Comment:
Please remove this-- it should be unnecessary because backbone-min.js is now part of base_vendor_js (loaded by main.html).